packages/perspective-viewer-d3fc/src/js/series/pointSeriesCanvas.js

```diff
@@ -43,11 +46,25 @@ export function pointSeriesCanvas(
             context.font = settings.textStyles.font;
             const { type } = settings.mainValues.find((x) => x.name === label);
             const value = toValue(type, d.row[label]);
-            const offset = size
-                ? Math.round(scale_factor * size(d.size)) * (15 / 1000) + 10
-                : 10;
 
-            context.fillText(value, offset, 4);
+            let magnitude = 10;
+
+            if (size) {
+                // `size(d.size)` is area (A) of circle.
+                // A = pi * r^2
+                // r = sqrt(A / pi)
+                const radius = Math.sqrt(
+                    (scale_factor * size(d.size)) / Math.PI
+                );
```

Comment:
This was news to me - I had assumed that the `.size()` method meant radius, but according to the docs you are indeed correct, [this is actually _area_](https://github.com/d3fc/d3fc/blob/master/packages/d3fc-series/README.md#seriesPoint_size)!